pkg/resourcemanager/webhook/endpointslicehints/add.go

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2022 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package endpointslicehints
+
+import (
+	discoveryv1 "k8s.io/api/discovery/v1"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+const (
+	// HandlerName is the name of this webhook handler.
+	HandlerName = "endpoint-slice-hints"
+	// WebhookPath is the path at which the handler should be registered.
+	WebhookPath = "/webhooks/default-endpoint-slice-hints"
```

Comment:
Similar to https://github.com/gardener/gardener/pull/7191#discussion_r1091878393, I added the default prefix because the webhook is defaulting one. The pod-schduler-name webhook's path is `/webhooks/default-pod-scheduler-name`, the seccomp-profile webhook's - without the default prefix, `/webhooks/seccomp-profile`.